docs/modules/ROOT/pages/reference/quality-portability.adoc

```diff
@@ -0,0 +1,45 @@
+= Portability
+
+== Services Run Independent of the Cloud They Are Provisioned
+Source::
+User
+
+Stimulus::
+Creates an AppCatalog Service
+
+Environment::
+AppCatalog enabled Kubernetes cluster
+
+Artifact::
+Kubectl or Service Broker API
+
+Response::
+The services are the same for all any AppCatalog enabled Kubernetes cluster zones.
+
+Response measure::
+The exception are resold Cloud services like RDS or CloudSQL.
+
+Rationale::
+The AppCatalog service does not care on what k8s they are running.
```

Comment:
Maybe I should rephrase it a bit, instead of Kubernetes we could say cloud. 

> Postgresql on Cloudscale is entirely different than on AWS.

Do you refer to the difference between Postgres via helm on k8s and RDS on AWS?

Reply:
IMHO this requirement targets how the service is provisioned. Ordering a Postgres instance on AWS should be the same as ordering it on cloudscale.ch.

Reply:
I could very well imagine such a case, yes.